lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -259,11 +280,16 @@ function getSapUiCoreBunDef(name, filters, preload) {
  * @param {module:@ui5/fs.DuplexCollection} parameters.workspace DuplexCollection to read and write files
  * @param {module:@ui5/fs.AbstractReader} parameters.dependencies Reader or Collection to read dependency files
  * @param {module:@ui5/builder.tasks.TaskUtil|object} [parameters.taskUtil] TaskUtil
+ * @param {string[]} [parameters.options.excludes=[]] List of modules declared as glob patterns (resource name patterns)
+ * that should be excluded. A pattern either contains of a trailing slash '/' or single '*' and double '**' asterisks
```

Comment:
same wording as in the generateComponentPreload.js

Reply:
See comment in generateComponentPreload